handle_wiki/extraction.py

```diff
@@ -27,18 +46,17 @@ def extractDataPerOccupation(refs: list, category: str, group: str) -> list:
         try:
             page = wptools.page(wikibase=ref.split("/")[-1])
             page.get_wikidata()
-            
             description = page.data["description"]
             title = page.data["title"]
 
             content = page.get_query().data["extext"].replace("\n", " ")
         except:
             traceback.print_exc()
             print(f"Can't obtain content for entry: {title}")
-            pass
+            continue 
         else:
             sentences = nltk.sent_tokenize(content)
-            print(title, len(sentences))
+            print(f"person name: {title}, sentences: {len(sentences)}")
```

Comment:
It was added simply for debugging purposes, so prettifying prints actually makes it less visible, imo.